### ISSUE TITLE
Enriching Quire (AHRC) Page: Institution Logos Not Displaying Correctly

### ISSUE THREAD

**Issue by bluebinary** [linked-art/linked.art#723]
While reviewing the new Linked.Art site page layout in Netlify, I noticed that the institution logos for the participant organizations included in the Enriching Quire (AHRC) project are not displaying correctly on the associated page; many of the logos are stretched, are showing at a very large size relative to the amount of space they are allocated in the page template, or are clipped due to overlapping the boundaries of the page: https://deploy-preview-701--linked-art.netlify.app/community/projects/ees2/

![image](https://github.com/user-attachments/assets/00083e4d-ca97-41dc-9978-e97a32d8d30f)

**Comment by azaroth42** [comment]
Good catch! Also fails for me on Chrome. I've commented out the images. If @krpage or @t-a-bonnet want to fix, please go ahead, but closing the issue for now.
